components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetMacros.xml

```diff
@@ -238,6 +238,31 @@
 #end
 #end
 
+#macro (__displayPropertyAsSelect $suggest_script)
+#if ($xcontext.action == 'edit' || $xcontext.action == 'inline')
+  {{html wiki=false clean=false}}
+    &lt;input name="xwiki-free-multiselect-suggest-script" type="hidden" value="$!escapetool.xml($suggest_script)"/&gt;&lt;input name="${prefix}${name}" type="hidden" value=""/&gt;
+  {{/html}}
+#end
+#macro (_fms__displayValue $v $counter)
+  #if($xcontext.action == 'edit' || $xcontext.action == 'inline')
+    {{html wiki=false clean=false}}
+      &lt;input class="xwiki-free-multiselect-value" type="text" size="30" name="${prefix}${name}" id="${prefix}${name}_${counter}" value="$!escapetool.xml($v)"/&gt;
+    {{/html}}
+  #else
+    $!{services.rendering.escape($v, $doc.syntax)}
+  #end
+#end
+(% class="xwiki-free-multiselect"%)
+#foreach ($v in $value)
+1. #_fms__displayValue($v ${foreach.count})
+
```

Comment:
Remove this newline, lists are numbered `1, 1, 1...` instead of `1, 2, 3...` because of it.